checkpoint_ngfw/plugin.spec.yaml

```diff
@@ -692,6 +692,50 @@ actions:
         type: boolean
         required: true
 
+  set_threat_protection:
+    title: Set Threat Protection
+    description: Set threat protection
```

Comment:
In light of some of the comments in the slack chat, Can you please expand on the description fields?  They are copies of the name field.